### PR TITLE
Add genetic code 25 to assemblySets

### PR DESCRIPTION
**Pull request by landml**
Stop giving warning about Prodigal and the unknown domain. kb_seed now handles this.
@JamesJeffryes

### CODE REVIEW

File: lib/RAST_SDK/RAST_SDKImpl.pm [hunk 1]
```diff
@@ -446,23 +446,19 @@ sub annotate_process {
...
+#			$inputgenome->{features} = [];
+			$message .= "The existing gene features were cleared due to selection of gene calling with Glimmer3 or Prodigal.\n";
+		}
+		if (length($genecalls) == 0) {
+			$genecalls = "Standard gene features were called using: ";
 		} else {
-			$message .= "Did not predict prodigal genes because the domain is $parameters->{domain}\n\n";	
+			$genecalls .= "; ";
+		}
+		$genecalls .= "prodigal";
```
Comment (JamesJeffryes) [inline lib/RAST_SDK/RAST_SDKImpl.pm:458]: This doesn't make too much sense to me. I assume it $genecalls becomes part of the report? Why append "; prodigal" if no genes get called?

Comment (landml) [reply lib/RAST_SDK/RAST_SDKImpl.pm:458]: This section of code is describing what the user selected. $genecalls is part of a longer message that is part of the messaging at the end. It is a list of all the tools that the user requested. The +/- statements in the commit make it hard to follow. I'm just putting these statements back they way there were a couple of commits ago and they parallel the glimmer section (lines 427-443). 

If the user selected prodigal but the $genecalls is still blank, prepend with "Standard gene features were called using: ";

Comment (JamesJeffryes) [reply lib/RAST_SDK/RAST_SDKImpl.pm:458]: 👍
